bin/alias.ml

```diff
@@ -43,7 +43,7 @@ let in_dir ~name ~recursive ~contexts dir =
     ; name
     ; contexts =
         [ List.find_exn contexts ~f:(fun c ->
-              Dune.Context_name.equal (Dune.Context.name c) ctx.name)
+              Dune.Context_name.equal (Dune.Context.name c) ctx.name )
```

Comment:
What's the rationale for this style? To my eyes, it looks strange to add a space before the ending paren, but not after the starting one.

Reply:
The idea is to add the space to distinguish the parentheses that have been opened/closed on the same line from the ones opened/closed in other lines.
The choice has been done for `fun` and `function` to only add the space before the closing parenthesis because the space after the opening parenthesis confused the users in this case.

Reply:
Sorry if this is getting OT, but out of curiosity (and not necessarily to use in this codebase), would it be possible to put the closing parenthesis in its own line?

Reply:
@nojb If/when https://github.com/ocaml-ppx/ocamlformat/pull/1377 is merged `indicate-multiline-delimiters` will put the closing `)` on its own line.